cumulusci/cumulusci.yml

```diff
@@ -6,6 +6,12 @@ tasks:
         group: Metadata Transformations
         description: Activates Flows identified by a given list of Developer Names
         class_path: cumulusci.tasks.salesforce.activate_flow.ActivateFlow
+    add_field_help_text:
```

Comment:
Should we rename this to `set_field_help_text` to match the Python class name?

Reply:
sure